examples/index.html

```diff
@@ -57,13 +59,13 @@
     <form>
       <label id="robots-2-label">Robots 2</label>
       <auto-complete src="/demo" for="items-2-popup" aria-labelledby="robots-2-label" >
-        <input name="robot" type="text" aria-labelledby="robots-2-label" autofocus>
+        <input name="robot-2" type="text" aria-labelledby="robots-2-label" autofocus>
         <ul id="items-2-popup" aria-labelledby="robots-2-label"></ul>
         <div id="items-2-popup-feedback" class="sr-only"></div>
       </auto-complete>
       <button type="submit">Save</button>
     </form>
-    <script type="module" src="./dist/bundle.js"></script>
-    <!-- <script type="module" src="https://unpkg.com/@github/auto-complete-element@latest/dist/bundle.js"></script> -->
+    <!-- <script type="module" src="./dist/bundle.js"></script> -->
+    <script type="module" src="https://unpkg.com/@github/auto-complete-element@latest/dist/bundle.js"></script>
```

Comment:
why was this change needed?

Reply:
This is for testing. Originally, the example used the published version, but I had commented it out to use the local. Either is fine. I can put comments to describe why one would be used over another if you think that'd be helpful. WDYT?